tests/every.spec.js

```diff
@@ -0,0 +1,22 @@
+import {every} from '../src';
+
+/* eslint-disable require-jsdoc */
+const testArray = [5, 12, 8, 130, 44];
+
+describe('every', () => {
+  it('should check each element matches the test conditions', () => {
+    const tests = [
+      (element) => Number.isInteger(element),
+      (element) => element < 30,
+      (element) => element > 0,
+      (element) => element == 'every',
+    ];
```

Comment:
I reckon these would best be served as their own tests. Each test spec is defined by the input to the unit that is being tested. In this instance, the inputs are the functions in the `tests` array and the unit is your `every` method.

I reckon two tests will be fine:
1. `it should return true for every element greater than 0`
1. `it should return false for every element less than 30`

Reply:
With the current approach, there are four instances in which the test can fail, making it harder to debug.